add docs for global index prefix (#3106)
astronomer/docs: software/release-notes.md

            enableSystemAdminCanUseAllRuntimes: true   
```

> ### changes to behaviour for configuring default index name prefix

Reviewer: ```suggestion
### Additional improvements
```
A change to behavior implies that there's something automatic, whereas this is a new feature that has to be explicitly configured.

> Until 0.33.1 we supported only hardcoded index prefix for both fluentd and sidecar based logging by defaulting to ```fluentd.*``` and ```vector.*```, this new configuration change allows users to configure global index name prefix to be used with both fluend and sidecar based logging

Reviewer: ```suggestion
- You can now configure a global index name prefix to use for both Fluentd and sidecar-based logging, which allows Astronomer Software to retain logs when you migrate between logging solutions. To enable this feature, add the following lines to your `config.yaml` file:
```

> ```yaml

Reviewer: ```suggestion
    ```yaml
```

    global:
       logging:
          indexNamePrefix: <your-index-prefix>

> ```

Reviewer: ````suggestion
    ```
````

> 
> Note: currently we don't support data migration. if a existing user migrates from fluentd to sidecar based logging he can utilize this feature by setting ```global.logging.indexNamePrefix: fluentd``` allowing platform to use common index to retain old task logs.

Reviewer: ```suggestion
```

Author: I thought it would allow existing users who are migrating to sidecar might give some insight on the feature

Reviewer: I removed this because I moved the same content into the note before the YAML block

> 
> ### Additional improvements
> 

Reviewer: ```suggestion
```

### Additional improvements

- You can now configure a global index name prefix to use for both Fluentd and sidecar-based logging, which allows Astronomer Software to retain logs when you migrate between logging solutions. To enable this feature, add the following lines to your `config.yaml` file:

    ```yaml
    global:
       logging:
          indexNamePrefix: <your-index-prefix>
    ```

> - Deprecated Kubernetes 1.23

Reviewer: ```suggestion
- Deprecated Kubernetes 1.23

```
